tyk-headless/templates/deployment-gw-repset.yaml

```
@@ -48,6 +48,14 @@ spec:
       affinity:
 {{ toYaml .Values.gateway.affinity | indent 8 }}
 {{- end }}
+      initContainers: 
+      - name: "setup-directories"
+        image: busybox
```

Always best to specify a version of an image to avoid unexpected behaviour in the future:
```suggestion
        image: busybox:1.32
```